src/App.js

```diff
@@ -1,28 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import BooksList from './containers/BooksList';
+import BooksForm from './containers/BooksForm';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <BooksList />
+      <BooksForm />
```

Comment:
Place this component inside `src/components` folder (it's a presentational component)

`"Create a components directory in the src folder for all the presentational components."`